src/modules/layout/components/Snackbar.js

```diff
@@ -39,9 +47,13 @@ const Action: StyledComponent<{}, ThemeType, *> = styled.Text`
 export type SnackbarActionType = {|
   label: string,
   onPress: () => void | Promise<void>
+|} | {|
+  label: string,
+  href: string
 |}
```

Comment:
It would be better to only have `label` and `onPress` here just like before. You can do the linking by passing an appropriate `onPress` method to Snackbar.